### DISCUSSION TITLE
Feature request: Allow copying of Font Awesome HTML classes on website

### DISCUSSION THREAD

**Discussion by ghost** [FortAwesome/Font-Awesome#20933] [Feature Requests]
### Is your feature request related to a problem?

Hey, the latest redesign of the Font Awesome website has made it impossible to just copy the Font Awesome classes that we need. Previously this was always possible but now the website only offers certain HTML snippets with an "i" class that are rarely relevant for our use-case. We are also not able to select and manually copy these classes anymore because the website will always copy the entire code snippet on click.

### Feature description

Ideally we would like to be able to easily copy the HTML classes on the website. For example:

https://fontawesome.com/icons/computer?s=solid

When visiting this page, I would like to be able to copy `fa-solid fa-computer` without all the extra HTML that is not relevant to our work. I would imagine a new button on the right-hand side which reads "Copy HTML classes" or something like that.

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by sensibleworld** [comment]
Thanks for the feedback! I'll bring this up with the team.
